src/assets/icons/amazon-prime.svg

```diff
@@ -1 +1 @@
-<svg viewBox="0 0 50 50" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M13.361 44.998a1.505 1.505 0 01-1.036-.347 1.366 1.366 0 01-.316-1.042V8.177c-.044-.376.07-.753.316-1.043a1.504 1.504 0 011.019-.295h3.897a1.384 1.384 0 011.51 1.164l.386 1.39a11.956 11.956 0 013.898-2.484 12.604 12.604 0 014.67-.904 10.07 10.07 0 018.217 3.787 15.84 15.84 0 013.055 10.23 18.425 18.425 0 01-1.51 7.73 12.181 12.181 0 01-4.09 5.088 10.197 10.197 0 01-5.935 1.737 12.728 12.728 0 01-4.214-.695 10.15 10.15 0 01-3.388-1.91v11.672a1.45 1.45 0 01-.299 1.042c-.3.229-.678.333-1.053.295l-5.127.017zm11.816-16.257a5.082 5.082 0 004.442-2.032 11.263 11.263 0 001.422-6.374 11.551 11.551 0 00-1.404-6.444 5.062 5.062 0 00-4.478-2.032 10.352 10.352 0 00-5.267 1.441v13.982a9.979 9.979 0 005.267 1.477l.018-.018z" fill="currentColor"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 17 17" fill="currentColor"><g fill-rule="evenodd"><path d="M14.506 12.63c-1.758 1.296-4.305 1.986-6.498 1.986-3.081 0-5.845-1.137-7.94-3.03-.159-.148-.016-.351.181-.235 2.26 1.316 5.06 2.107 7.943 2.107 2.082-.01 4.141-.43 6.06-1.239.297-.126.547.195.254.41zm.731-.837c-.223-.287-1.485-.135-2.05-.069-.173.021-.199-.129-.044-.237 1.01-.706 2.653-.503 2.846-.265.192.237-.05 1.89-.994 2.68-.145.12-.283.056-.212-.105.212-.529.687-1.716.462-2.004M9.585 6.75c0 .579.015 1.062-.291 1.577-.251.417-.645.675-1.077.675-.598 0-.948-.435-.948-1.077 0-1.267 1.189-1.498 2.316-1.498v.322zm1.57 3.625a.337.337 0 0 1-.368.035c-.516-.41-.61-.6-.893-.991-.854.832-1.46 1.081-2.565 1.081C6.018 10.5 5 9.728 5 8.182c0-1.207.683-2.03 1.66-2.431.844-.356 2.024-.418 2.925-.514V5.05c0-.354.03-.772-.19-1.078-.188-.274-.551-.387-.873-.387-.593 0-1.12.291-1.25.894-.026.137-.129.265-.27.272l-1.514-.162c-.126-.027-.268-.125-.231-.308C5.606 2.528 7.26 2 8.74 2c.758 0 1.747.193 2.344.74.758.676.685 1.578.685 2.56v2.318c0 .697.303 1.002.587 1.379.099.137.12.294-.006.395-.402.324-.8.654-1.193.987l-.002-.004"/></g></svg>
```

Comment:
Don't fill the background, remove the `fill` in `<svg` but leave the one in `<path`. This way it'd have a background that we are trying to avoid.